frontends/precinct-scanner/src/screens/admin_screen.tsx

```diff
@@ -179,6 +210,34 @@ export function AdminScreen({
           </div>
         </Bar>
       </Absolute>
+      {isShowingToggleLiveModeWarningModal && (
```

Comment:
Very minor! This boolean's name made me expect a modal that would prevent switching to live mode when it's actually preventing the reverse. Something like `isBackupBeforeTogglingTestModeWarningModalOpen` might be clearer (though a bit verbose)